libr/core/cmd_search.c

```diff
@@ -2055,22 +2045,9 @@ static void do_asm_search(RCore *core, struct search_parameters *param, const ch
 		json = 0;
 	}
 
-	r_list_free (param->boundaries);
-	param->boundaries = r_core_get_boundaries_prot (core, -1, param->mode, "search");
 	maxhits = (int) r_config_get_i (core->config, "search.maxhits");
 	filter = (int) r_config_get_i (core->config, "asm.filter");
 
-	if (!param->boundaries) {
-		map = R_NEW0 (RIOMap);
-		if (map) {
-			map->fd = r_io_fd_get_current (core->io);
-			map->itv.addr = r_config_get_i (core->config, "search.from");
-			map->itv.size = r_config_get_i (core->config, "search.to") - map->itv.addr;
```

Comment:
Is search.from/to honored in params already?

Reply:
My thinking was its captured by search_itv